SwissDataScienceCenter/renku-ui
# fix: error when notebook doesnt have value

PR to fix error when open notebook.
Solution: Process Markdown only if it contain valid props.

#### How to  test:
Open https://renku-ci-ui-1926.dev.renku.ch/projects/cramakri/inflation/files/blob/qtm/QTM-Part1.ipynb and its equivalent in gitlab https://dev.renku.ch/gitlab/cramakri/inflation/blob/master/qtm/QTM-Part1.ipynb the content displayed should be the same

closes #1906 

/deploy renku=tests-ui-2.5.0 #persist
